Frends.LDAP.SearchObjects/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [2.0.0] - 2024-11-06
+### Added
+- [Breaking] Parameter for AnonymousBind to enable to connect without credentials.
```

Comment:
Add a short description under breaking changes on how to upgrade, if someone already uses the previous version. E.g. set AnonymousBind to false and such. And for the version field mention which version we used before, so it is clear also how to make sure that the task behaves the same way as before.